### ISSUE TITLE
**bugs** does not highlight the books in archive.org 

### ISSUE THREAD

**Issue by JJger148** [hypothesis/h#7014]
### Expected behaviour
The text in archive.org should be highlighted and the highlighted text in hypothes.is should link back to the boos in archive.org

### Actual behaviour
text in archive.org is not highlighted only the sentences are copied, but they do not link back. 

### Browser/system information
Mac M1
Brave Browser

### Additional details
maybe it is an issue from openlibrary? 
https://openlibrary.org/dev/docs/bookreader#contributors

**Comment by robertknight** [comment]
Can you provide an example URL that shows the problem you are referring to? Can you also clarify what mechanism you are trying to use to add Hypothesis to the page (browser extension, Via, bookmarklet)?

**Comment by robertknight** [comment]
Looking at https://archive.org/details/b22382343 using the browser extension, I see that the text layer on the page is created by an SVG. The Hypothesis client doesn't currently support annotating text in SVG images. Additionally the book viewer is a single page application. It does update the URL as you navigate through the book, but Hypothesis doesn't currently react by changing the set of loaded annotations.

In summary, the minimum set of changes that would be required to make this work are:

- Support highlighting text in SVG graphics
- Support changing loaded set of annotations when navigating in single-page applications

There might be other things as well.
